andela/travel_tool_front
# #160597069 Add guest room and bed during request

#### What does this PR do?

This PR ensure that user enters available room and bed space during new travel request creation.

#### Description of Task to be completed?

As a user, I should be able to add the room and bed space during my new travel request. This will ensure that if I will not get any room available, they will look for alternatives or postpone trip duration. 
It makes sure that room selected is based on user destination, gender and travel duration. 

#### How should this be manually tested?

- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Change into the project directory
- git checkout ft-check-available-room-bed-160597069
- Run the server on [this PR](https://github.com/andela/travel_tool_back/pull/92)
- Login and make a new request
- Make three trips, ( oneWay, Multi and return ) trips
- After filling the details on making a new request, when checking for available rooms, the room name and bed name available are shown. 

#### Any background context you want to provide?
Ensure that you have guest houses that has rooms and beds. 

#### What are the relevant pivotal tracker stories?

[#160597069](https://www.pivotaltracker.com/story/show/160597069)

#### Screenshots (if appropriate)

<img width="1417" alt="screen shot 2018-10-18 at 17 03 27" src="https://user-images.githubusercontent.com/32410518/47160053-c6de1b80-d2f7-11e8-8aa9-b8179a4c5b41.png">

#### Questions:
